ui-admin/src/study/participants/survey/SurveyResponseView.tsx

```diff
@@ -123,11 +126,17 @@ export function RawEnrolleeSurveyView({
   const [autosaveStatus, setAutosaveStatus] = useState<AutosaveStatus | undefined>()
   const [showJustificationModal, setShowJustificationModal] = useState(false)
   const [justification, setJustification] = useState<string>('')
+  const [showTaskModal, setShowTaskModal] = useState(false)
 
   return <div>
     <div>
       <div className="d-flex align-items-center justify-content-between">
-        <div>{surveyTaskStatus(response)}</div>
+        <div className="d-flex align-items-center">
+          {surveyTaskStatus(task, response)}
+          <Button variant="secondary" onClick={() => setShowTaskModal(!showTaskModal)}>
+            change / remove
+          </Button>
```

Comment:
ux thought: the top row in the survey response view is getting to be a bit busy. i wonder if we can roll this action button into the status badge itself. this is similar to what we do for editing participant note assignment (although i think this is an improvement over that UX)

something like this:
<img width="1261" alt="Screenshot 2024-11-20 at 3 39 47 PM" src="https://github.com/user-attachments/assets/822e6f72-d502-4d1a-be24-ec29d6b87e48">

i threw the changes up on the branch `mb-edit-status-ux-tweak` if you want to pull it in. it was easier than trying to use a code suggestion comment

Reply:
Love this! Another idea that thought could be cool if the status was a dropdown (basically turn the pencil into an arrow) and when you selected something other than the current task status it would prompt for justification.